src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -955,11 +958,11 @@ end⟩
 lemma connected_of_edge_connected {k : ℕ} (hk : 0 < k) (h : G.edge_connected k) :
   G.connected :=
 begin
-  intros v w,
+  intros v w, --breaking here
```

Comment:
Not sure how to proceed here

Reply:
Your proposition is now a conjunction of two propositions. So you need to "split" then apart. You'll get two goals.

Reply:
Yeah, one of them is `nonempty V`, but I suspect we don't have how to prove it. I think (I could very much be wrong) that we have to add this extra restriction to make this lemma work.